src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -66,5 +66,10 @@ def get_from_project(self, project_id: int) -> List[APIUser]:
     def get_by_actor_ids(self, *, actor_ids: List[int]) -> List[APIUser]:
         return [UserService.serialize_user(u) for u in User.objects.filter(actor_id__in=actor_ids)]
 
+    def get_participating_users(self, group: Group) -> List[APIUser]:
+        return list(
+            User.objects.filter(groupsubscription__is_active=True, groupsubscription__group=group)
+        )
```

Comment:
This is returning `User`, not `APIUser`. They need to be serialized before being returned from the service method, as in

```suggestion
        return [
            self.serialize_user(user)
            for user in User.objects.filter(
                groupsubscription__is_active=True, groupsubscription__group=group
            )
        ]
```

Alternatively, you might be able to use `serialize_users`, but that would probably result in a redundant DB query.

Reply:
i'm going to add an additional param to serialize_users so you can pass in a list of users to serialize

Reply:
scratch that -- it looks like there were updates to the endpoint that make it region stable without me having to do anything